src/server/game/Server/Packets/InstancePackets.h

```diff
@@ -236,6 +236,15 @@ namespace WorldPackets
             int32 InCombatResCount = 0;
             uint32 CombatResChargeRecovery = 0;
         };
+
+        class InstanceBossKillCredit final : public ServerPacket
```

Comment:
also please rename this class to BossKillCredit as in SMSG_BOSS_KILL_CREDIT (there is no INSTANCE word here)